sunpy/net/dataretriever/client.py

```diff
@@ -18,6 +23,10 @@
 __all__ = ['QueryResponse', 'GenericClient']
 
 
+
+def simple_path(path, sock, url):
+    return path
```

Comment:
the Downloader expects a callable with (sock, url) arguments, so this is used in the get method.